<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rechercheDialog</class>
 <widget class="QWidget" name="rechercheDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>558</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>558</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>517</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0" colspan="2">
     <widget class="QCheckBox" name="chkPhoto">
      <property name="text">
       <string>Annonces contenant photo</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelSuperficie">
      <property name="text">
       <string>Superficie entre (min/max)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="superficieMin"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelType">
      <property name="text">
       <string>Type du bien</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkTerrain">
        <property name="text">
         <string>Terrain</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkFerme">
        <property name="text">
         <string>Ferme</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkBureau">
        <property name="text">
         <string>Bureau</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chkAutre">
        <property name="text">
         <string>Autre</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelCodePostal">
      <property name="text">
       <string>Code Postal</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="0" colspan="2">
       <widget class="QDateEdit" name="dateAnnonce"/>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioDateApres">
        <property name="text">
         <string>Après</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioDateAvant">
        <property name="text">
         <string>Avant</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="piecesMax">
      <property name="maximum">
       <number>99999999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="prixMax">
      <property name="maximum">
       <number>99999999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="piecesMin"/>
    </item>
    <item row="4" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkMaison">
        <property name="text">
         <string>Maison</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkChateak">
        <property name="text">
         <string>Château</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkAppt">
        <property name="text">
         <string>Appartement</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="ckCommerce">
        <property name="text">
         <string>Commerce</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelPrix">
      <property name="text">
       <string>Prix entre (min/max)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="superficieMax">
      <property name="maximum">
       <number>99999999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelPieces">
      <property name="text">
       <string>Pièces entre (min/max)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="prixMin"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelDate">
      <property name="text">
       <string>Date de l'annonce</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="codePostal"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>Recherche d'une annonce</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
